hudi-common/src/main/java/org/apache/hudi/avro/ConvertingGenericData.java

```diff
@@ -57,12 +57,20 @@ private ConvertingGenericData() {
     addLogicalTypeConversion(DATE_CONVERSION);
     addLogicalTypeConversion(TIME_MICROS_CONVERSION);
     addLogicalTypeConversion(TIMESTAMP_MICROS_CONVERSION);
-    // NOTE: Those are not supported in Avro 1.8.2
-    // TODO re-enable upon upgrading to 1.10
-    // addLogicalTypeConversion(TIME_MILLIS_CONVERSION);
-    // addLogicalTypeConversion(TIMESTAMP_MILLIS_CONVERSION);
-    // addLogicalTypeConversion(LOCAL_TIMESTAMP_MILLIS_CONVERSION);
-    // addLogicalTypeConversion(LOCAL_TIMESTAMP_MICROS_CONVERSION);
+    // NOTE: Those are not supported in Avro 1.8.2 (used by Spark 2)
+    // Only add conversions if they're available
```

Comment:
Should we validate the fix and added tests with spark 2? I am not sure if CI covers it by default.